### DISCUSSION TITLE
How to customize colors for headings?

### DISCUSSION THREAD

**Discussion by hopezh** [squidfunk/mkdocs-material#7031] [Q&A] [answered]
May I ask how to specify different colors for different levels of headings in mkdocs? 

I'm asking this because I'd like to differentiate the headings from body text by not only font size but also by color. 

Thanks.

**Comment by alexvoss** [comment, marked as answer]
All you need to do is to add some [custom CSS](https://squidfunk.github.io/mkdocs-material/customization/?h=extra+css#additional-css) to your project. You will want the CSS selector to be specific enough to override any given styling.

```css
.md-content h1 { color: red; }
```

should be enough. Do the same for H2, H3 etc. headings.
